packages/components/src/components/gradient-background/gradient-background.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { CSSProperties } from 'react';
+import './gradient-background.scss';
+
+type TProps = {
+    color: CSSProperties['backgroundColor'];
+    primary: CSSProperties['background'];
+    secondary: CSSProperties['background'];
+    tertiary: CSSProperties['background'];
+};
+
+const GradientBackground: React.FC<React.PropsWithChildren<TProps>> = ({
+    children,
+    color,
+    primary,
+    secondary,
+    tertiary,
+}) => (
+    <div className='gradient-background'>
+        <div className='gradient-background__container' style={{ backgroundColor: color }}>
```

Comment:
```suggestion
    <div className='gradient-background' style={{ backgroundColor: color }}>
        <div className='gradient-background__container' >
```
hey @farzin-deriv we need to add background to parent container. i have one use case whit Demo background. it's kind of tricky. because it uses black background in container in light mode and the rest of left and right areas stay white. it's a little bit hard to explain by words...